### PR TITLE
Update from Weblate

### PR DESCRIPTION
**Pull request by xwikiorgci**

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_es.properties [hunk 1]
```diff
@@ -2768,11 +2768,11 @@ xe.panels.rights.users=Usuarios
 xe.panels.rights.usersexplanation=Este campo deber\u00EDa contener el identificador wiki de cada usuario al que desee aplicar los derechos. Por ejemplo, para usuarios no identificados deber\u00EDa usar XWiki.JohnDoe. XWiki.XWikiGuest
 xe.panels.rights.groups=Grupos
 xe.panels.rights.groupsexplanation=Este campo deber\u00EDa contener el nombre Wiki de cada grupo a los que desea que se le apliquen los derechos. XWiki.XWikiAllGroup representa el grupo de todos los usuarios con una cuenta en su Wiki que est\u00E1n autenticados.
-xe.panels.rights.groupsvirtualexplanation=xwiki:XWiki.XWikiAllGroup representa al grupo de todos los usuarios aut\u00E9nticados utilizando una cuenta global.
+xe.panels.rights.groupsvirtualexplanation=<em>xwiki:XWiki.XWikiAllGroup</em> representa el grupo de todos los usuarios autenticados que utilizan una cuenta global.
```
Comment (vmassol) [inline xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_es.properties:2771]: Not sure it's good to put HTML in translations. I don't think that's correct.

File: xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_es.properties [hunk 2]
```diff
@@ -2768,11 +2768,11 @@ xe.panels.rights.users=Usuarios
...
 xe.panels.rights.accesslevelsexplanation=Este campo debe contener una lista de los niveles de acceso que se quieren aplicar a los usuarios y grupos especificados. Los niveles de acceso disponibles son: admin (administraci\u00F3n), programming (proramaci\u00F3n), register (registro), edit (edici\u00F3n), view (visualizaci\u00F3n), comment (comentarios). Para proteger su Wiki en modo vista y edici\u00F3n utilice "view, edit". Para proteger de a\u00F1adir comentarios utilice "comment".
 xe.panels.rights.allowdeny=Permitir/Denegar
-xe.panels.rights.allowdenyexplanation=Este campo deber\u00EDa contener 'Permitir' para indicar que se dispone del derecho y 'Denegar' para indicar que no se dispone de este derecho. Si se dispone del derecho el Wiki o el Espacio es *exclusivamente* visible o editable para los usuarios o grupos que disponen del derecho.
+xe.panels.rights.allowdenyexplanation=Este campo deber\u00EDa contener <em>Permitir</em> para indicar que se dispone del derecho, y <em>Denegar</em> para indicar que no se dispone de ese derecho. Un derecho <em>permitido</em> significa: "esta wiki, espacio o p\u00E1gina es *solo" visible o editable por los usuarios o grupos especificados".
```
Comment (vmassol) [inline xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_es.properties:2775]: Same comment

**Comment by vmassol** [comment]
@committers: WDYT about HTML in the translations above?

**Comment by vmassol** [comment]
We need to document if it's allowed or not here :https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HTranslationBestPractices

**Comment by vmassol** [comment]
Actually we use HTML in existing translations (like for BR or A) but I think it's not right to use HTML to highlight text such as EM above. WDYT? Using `<tt>` is probably acceptable.

File: xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_es.properties (hunk 1 above)
Comment (tmortagne) [reply xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_es.properties:2771]: It's not great but that's how this key was designed according to the default version so not much to do with this translations.

File: xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_es.properties (hunk 2 above)
Comment (tmortagne) [reply xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_es.properties:2775]: Same here.

**Comment by surli** [comment]
Closing the PR, there's something very wrong: apparently some files to merge are using a path with `.translation` which is a technical path from Weblate. I need to check why they are committed like that on Weblate.

**Pull request closed without merging**